src/components/molecules/message/message.html

```diff
@@ -0,0 +1,5 @@
+<div class="Message">
```

Comment:
```suggestion
<div class="message">
```
サイト制作課題では、CSSのクラス名は基本的にケバブケースで書いていきましょう！
参考：https://qiita.com/ybiquitous/items/75288bacb596a82a2805